static_build/scss/components/_day_picker.scss

```diff
@@ -0,0 +1,148 @@
+/* DayPicker styles */
+$day-picker-padding: 10%;
+$calendar-row-height: 40px;
+$calendar-day-size: 30px;
+
+.DayPicker {
```

Comment:
wondering why we are using capitalized-camel-cased form for our class name. is it a semantic ui thing? I would have expected `.dayPicker` or `.day-picker` not `.DayPicker`